tick-api/src/main/java/me/aurium/tick/driver/TickDriver.java

```diff
@@ -0,0 +1,54 @@
+package me.aurium.tick.driver;
+
+import me.aurium.tick.container.container.JDBCContainer;
+
+import java.sql.*;
+import java.util.Properties;
+import java.util.Set;
+import java.util.logging.Logger;
+
+public class TickDriver implements Driver {
+
+    private final Set<JDBCContainer> containerSet;
+
+    public TickDriver(Set<JDBCContainer> containerSet) {
+        this.containerSet = containerSet;
+
+        //(move this elswhere duh) DriverManager DriverManager.registerDriver(new TickDriver);
```

Comment:
Usually, a few things happen:
1. registerDriver it is placed inside the static initializer for the class
2. The constructor of the driver is not public
3. Service loader entries are added for the driver. The current standard is to do this in module-info